core/src/main/java/org/jdbi/v3/core/mapper/reflect/InstanceFactory.java

```diff
@@ -44,10 +46,10 @@ Parameter[] getParameters() {
         return executable.getParameters();
     }
 
-    Type[] getTypes() {
+    List<Type> getTypes() {
```

Comment:
same here - should it be `Type getType(int index)`?

Reply:
OK by me